libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -229,6 +229,13 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
 #endif
 #endif  // HAL_BARO_WIND_COMP_ENABLED
 
+    // @Param: BARO_POS
```

Comment:
```suggestion
#if AP_BARO_POSITION_COMPENSATION_ENABLED
    // @Param: BARO_POS
```

... this should be a zero-compiler-change output on 1MB boards.  And definitely off on AP_Periph.